prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -70,6 +70,21 @@ public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions s
             }
         }
 
+        // GET: api/Enrollees/PotentialPaperEnrolleeReturneeStatus/5
```

Comment:
There are some issues with this URL. In general, our URLs should be lower case and use hyphens instead of camel casing. As well, this is *really* wordy; I would prefer to see something shorter.

Reply:
Changed to HEAD and shortened the name and used hyphens instead of camel case